README.md

```diff
@@ -172,6 +174,33 @@ For Android: add the following line in your AndroidManifest.xml
 ```
 For IOS: configure [App Transport Security](https://developer.apple.com/library/content/documentation/General/Reference/InfoPlistKeyReference/Articles/CocoaKeys.html#//apple_ref/doc/uid/TP40009251-SW33) in your app
 
+#### Tile Overlay using local tiles
+
+Tiles can be stored locally within device using xyz tiling scheme and displayed as tile overlay as well. This is usefull especially for offline map usage when tiles are available for selected map region within device storage.
+
+```jsx
+<MapView
+  region={this.state.region}
+  onRegionChange={this.onRegionChange}
+>
+  <MapView.LocalTile
+   /**
+    * The path template of the locally stored tiles. The patterns {x} {y} {z} will be replaced at runtime
+    * For example, /storage/emulated/0/mytiles/{z}/{x}/{y}.png
+    */
+   pathTemplate={this.state.pathTemplate}
+   /**
+    * The size of provided local tiles (usually 256 or 512).
+    */
+   tileSize={256}
+  />
+</MapView>
+```
+
+For Android: if original (for example Google) tiles are not desirable (no need to download them when using offline tiles), set mapType to 'none'.
```

Comment:
not sure what you mean in this phrase :)

Reply:
Well, if you add LocalTile overlay into the map in Android, you can still see that google map loads tiles underneath your 'offline' tiles (if device is online). That can go against the whole reason why offline tiles are used in the first place.

Reply:
Does mapType={"none"} help?

Reply:
@zavadpe maybe you should explain that in the file like you did for me? 😊

Reply:
Yeah, it helped me in my project :) I'm using this snippet:
mapType={Platform.OS == "android" ? "none" : "standard"}

Reply:
@rborn ok, I'll add more thorough explanation ;)

Reply:
Yes, `mapType={Platform.OS == "android" ? "none" : "standard"}` please add it to the docs to avoid future issues like those mentioned in #1880.

BTW I was thinking about putting to roadmap android support for OSM, just for this cases #773, what do you think?

Reply:
@alvelig But are you thinking to add OSM support within google MapView or something separate? If not google than [osmdroid](https://github.com/osmdroid/osmdroid) can be of huge help/inspiration.

Reply:
The point is to replace GoogleMaps not to be bound to GMaps license agreement. Not saying that we will do it right now, but just a way to go.

Reply:
@alvelig Get it, sounds like fun :)